packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -38,25 +41,122 @@ def ensure_routable(cmd, service_points):
 
 
 class VipTest:
-    def __init__(self, num, vip, vipaddr, samehost, vipnet, proxynet):
+    def __init__(self, num, container, vip, vipaddr, samehost, vipnet, proxynet):
+        self.num = num
         self.vip = vip.format(num, 7000 + num)
         self.vipaddr = vipaddr.format(num, 7000 + num)
+        self.container = container
         self.samehost = samehost
         self.vipnet = vipnet
         self.proxynet = proxynet
+        self.notes = ""
 
     def __str__(self):
-        return ("VipTest(vip={},vipaddr={},samehost={},"
-                "vipnet={},proxynet={})").format(self.vip, self.vipaddr, self.samehost, self.vipnet, self.proxynet)
+        return ('VipTest(container={}, vip={},vipaddr={},samehost={},'
+                'vipnet={},proxynet={}, notes={})').format(self.container, self.vip, self.vipaddr, self.samehost,
+                                                           self.vipnet, self.proxynet, self.notes)
 
     def log(self, s, lvl=logging.DEBUG):
         m = 'VIP_TEST {} {}'.format(s, self)
         log.log(lvl, m)
 
 
+def docker_vip_app(network, host, vip):
```

Comment:
I meant, adding a newline at some places in the code, and presenting the logical chunks together improved the readability.

this diff:


```
diff --git a/packages/dcos-integration-test/extra/test_networking.py b/packages/dcos-integration-test/extra/test_networking.py
index a146a6c9..2fed4816 100644
--- a/packages/dcos-integration-test/extra/test_networking.py
+++ b/packages/dcos-integration-test/extra/test_networking.py
@@ -68,6 +68,7 @@ def docker_vip_app(network, host, vip):
     app['container']['docker']['network'] = network
     app['mem'] = 16
     app['cpu'] = 0.01
+
     if network == 'HOST':
         app['cmd'] = '/opt/mesosphere/bin/dcos-shell python '\
                      '/opt/mesosphere/active/dcos-integration-test/util/python_test_server.py $PORT0'
@@ -112,7 +113,9 @@ def mesos_vip_app(num, network, host, vip, ucr=False):
         'timeoutSeconds': 10,
         'maxConsecutiveFailures': 3,
     }]
+
     assert network != 'BRIDGE'
+
     if network == 'HOST':
         app['cmd'] = '/opt/mesosphere/bin/dcos-shell python '\
                      '/opt/mesosphere/active/dcos-integration-test/util/python_test_server.py $PORT0'
@@ -123,6 +126,7 @@ def mesos_vip_app(num, network, host, vip, ucr=False):
         if vip is not None:
             app['portDefinitions'][0]['labels'] = {'VIP_0': vip}
         app['healthChecks'][0]['portIndex'] = 0
+
     if network == 'USER':
         app['ipAddress'] = {
             'discovery': {
@@ -140,8 +144,11 @@ def mesos_vip_app(num, network, host, vip, ucr=False):
             app['ipAddress']['discovery']['ports'][0]['labels'] = {'VIP_0': vip}
         app['healthChecks'][0]['port'] = port
         app['portDefinitions'] = []
+
     app['constraints'] = [['hostname', 'CLUSTER', host]]
+
     log.info('app: {}'.format(json.dumps(app)))
+
     return app, uuid


@@ -189,6 +196,7 @@ def vip_test(dcos_api_session, r):
         cmd = '/opt/mesosphere/bin/curl -s -f -m 5 http://{}/test_uuid'.format(r.vipaddr)
         returned_uuid = ensure_routable(cmd, proxy_host, proxy_port)
         log.debug('returned_uuid is: {}'.format(returned_uuid))
+
     assert returned_uuid is not None
     assert returned_uuid['test_uuid'] == app_uuid
     r.log('PASSED')
```